aurora/model/aurora.py

```diff
@@ -665,6 +670,21 @@ def __init__(
         atmos_static_vars: bool = True,
         separate_perceiver: tuple[str, ...] = ("co", "no", "no2", "go3", "so2"),
         modulation_head: bool = True,
+        predict_difference_history_dim_lookup={
```

Comment:
This appears to be a duplicate of the variable in the class definition. Should we retain only one?